webui/react/src/pages/ExperimentDetails/ExperimentChart.tsx

```diff
@@ -43,6 +43,7 @@ const ExperimentChart: React.FC<Props> = ({ validationMetric, ...props }: Props)
       hovermode: 'y unified',
       hovertemplate: '%{text}<extra></extra>',
       mode: 'lines+markers',
+      name: validationMetric,
```

Comment:
looks like `validationMetric` could be undefined but if that's the case we probably don't have validation history an thus any lines either so we should be fine.